docs/resources/Conformance-as-Postman.postman_collection.json

```diff
@@ -54,11 +53,12 @@
 					}
 				},
 				"url": {
-					"raw": "{{baseUrl}}/studies",
+					"raw": "{{baseUrl}}/v{{version}}/studies",
```

Comment:
same here

Reply:
Added  a note to the instructions for using postman and changed the default of baseUrl to include the version